cookbooks/letsencrypt/recipes/default.rb

```diff
@@ -0,0 +1,48 @@
+letsencrypt = node['letsencrypt']
+
+package "certbot" do
+  action :install
+end
+
+md = fetch_env_var(node, 'EY_LE_MAIN_DOMAIN').downcase
+domain = fetch_env_var(node, 'EY_LE_DOMAINS').nil? || (fetch_env_var(node, 'EY_LE_DOMAINS').gsub(" "," -d ")).downcase
+app = fetch_env_var(node, 'EY_LE_MAIN_APP_NAME') || node['dna']['applications'].keys.first
+wc = fetch_env_var(node, 'EY_LE_USE_WILDCARD') || false
+
+if Dir.exist?("/data/#{app}/current") && ['solo', 'app_master'].include?(node['dna']['instance_role'])
+ 
+  execute "force start haproxy / nginx" do
```

Comment:
@mushyy why do we need a hard restart here? 
@paulasaurus will this restart trigger a takeover? my take is "yes, it will"....

Reply:
Its a force start not a force restart. Its to ensure nginx / haproxy is running so the certificate can be authenticated when it writes the challenge file.

Reply:
have we found HAProxy / Nginx to not be running in some cases? or just added as a preventive messure?

Reply:
Preventive measure